LayoutTests/storage/indexeddb/modern/resources/idbdatabase-transaction-failures.js

```diff
@@ -31,8 +27,8 @@ function prepareDatabase(event)
     }
 
     versionTransaction.onabort = function(event) {
-        debug("versionchange transaction aborted");
-        done();
+        // Set event handler to null to avoid unexpected error message being printed.
```

Comment:
Should event handler be set to null here?

Reply:
ah, actually I've moved the setting to line 12 -- I need to get rid of the comment here.